packer/bastion.json

```diff
@@ -1,59 +1,75 @@
 {
-    "builders": [{
-        "type": "amazon-ebs",
-        "region": "us-east-2",
-        "source_ami_filter": {
-            "filters": {
-                "virtualization-type": "hvm",
-                "name": "debian-stretch-hvm-x86_64-gp2-*",
-                "root-device-type": "ebs"
-            },
-            "owners": [
-                "379101102735"
-            ],
-            "most_recent": true
-        },
-        "instance_type": "t3.micro",
-        "ssh_username": "admin",
-        "ami_name": "cyhy-bastion-hvm-{{timestamp}}-x86_64-ebs",
-        "ami_regions": [
-            "us-east-1",
-            "us-west-1",
-            "us-west-2"
-        ],
-
-        "ami_block_device_mappings": [{
-            "device_name": "xvda",
-            "volume_size": 8,
-            "volume_type": "gp2",
-            "encrypted": true,
-            "delete_on_termination": true
-        }],
-
-        "launch_block_device_mappings": [{
-            "device_name": "xvda",
-            "volume_size": 8,
-            "volume_type": "gp2",
-            "encrypted": true,
-            "delete_on_termination": true
-        }],
-
-        "tags": {
-            "Team": "NCATS OIS - Development",
-            "Application": "Cyber Hygiene",
-            "OS_Version": "Debian Stretch",
-            "Release": "Latest",
-            "Base_AMI_Name": "{{ .SourceAMIName }}"
+  "builders": [
+    {
+      "ami_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "/dev/xvda",
+          "encrypted": true,
+          "volume_size": 8,
+          "volume_type": "gp2"
         }
-    }],
-
-    "provisioners": [
+      ],
+      "ami_name": "cyhy-bastion-hvm-{{timestamp}}-x86_64-ebs",
+      "ami_regions": [
+        "us-east-1",
+        "us-west-1",
+        "us-west-2"
+      ],
+      "instance_type": "t3.micro",
+      "launch_block_device_mappings": [
         {
-            "type": "ansible",
-            "playbook_file": "packer/ansible/playbook.yml",
-            "groups": [
-                "bastion"
-            ]
+          "delete_on_termination": true,
+          "device_name": "/dev/xvda",
+          "encrypted": true,
+          "volume_size": 8,
+          "volume_type": "gp2"
         }
-    ]
+      ],
+      "region": "us-east-2",
+      "source_ami_filter": {
+        "filters": {
+          "name": "debian-10-amd64-*",
+          "root-device-type": "ebs",
+          "virtualization-type": "hvm"
+        },
+        "most_recent": true,
+        "owners": [
+          "136693071363"
+        ]
+      },
+      "ssh_username": "admin",
+      "tags": {
+        "Application": "Cyber Hygiene",
+        "Base_AMI_Name": "{{ .SourceAMIName }}",
+        "OS_Version": "Debian Buster",
+        "Release": "Latest",
+        "Team": "NCATS OIS - Development"
```

Comment:
As long as we are updating our Packer JSON files, we might as well update our team tag to the slightly more modern `VM Fusion - Development` that we use elsewhere (e.g. [kali-packer](https://github.com/cisagov/kali-packer/blob/develop/src/packer.json#L54)).

This change should be made to all of our Packer files in this repo.

Reply:
Good catch! I did a find/replace on the repo in 3f8e4c6.